### PR TITLE
Bugfix: Fix potential crash when renaming custom buttons

### PR DESCRIPTION
**Pull request by wutschel**
## Description

Fixes a sporadic issue reported via AppStore by early return to avoid out-of-bounds-access.

## Summary for release notes

Bugfix: Fix potential crash when renaming custom buttons

### CODE REVIEW

File: XBMC Remote/RightMenuViewController.m [hunk 1]
```diff
@@ -444,18 +444,21 @@ - (void)tableView:(UITableView*)tableView accessoryButtonTappedForRowWithIndexPa
         textField.text = tableData[indexPath.row][@"label"];
     }];
     UIAlertAction *updateButton = [UIAlertAction actionWithTitle:LOCALIZED_STR(@"Update label") style:UIAlertActionStyleDefault handler:^(UIAlertAction *action) {
+        if (indexPath.row >= tableData.count || !alertView.textFields.count) {
```
Comment (kambala-decapitator) [inline XBMC Remote/RightMenuViewController.m:447]: the second condition will always be false

Comment (wutschel) [reply XBMC Remote/RightMenuViewController.m:447]: Does `textFields` guarantee to have at least 1 item in the array?

Comment (kambala-decapitator) [reply XBMC Remote/RightMenuViewController.m:447]: well, you're adding a text field a few lines above :) so yes, they don't disappear on their own

Comment (wutschel) [reply XBMC Remote/RightMenuViewController.m]: And there yo got me. Changed.

**Pull request merged**